app/models/jobseeker.rb

```diff
@@ -1,4 +1,7 @@
 class Jobseeker < ApplicationRecord
   devise :database_authenticatable, :registerable, :recoverable, :validatable,
          :confirmable, :lockable, :trackable, :timeoutable
+
+  has_many :jobseeker_saved_vacancies, dependent: :destroy
+  has_many :saved_jobs, through: :jobseeker_saved_vacancies, source: :vacancy
```

Comment:
`saved_jobs, through: :jobseeker_saved_vacancies`

Not sure but we should define just one word everywhere. Either `job` or `vacancy`. Or maybe `ad`?

Reply:
I had about 500 different names for the join table 😂

Reply:
+1 for `jobseeker_saved_jobs`

Reply:
The reason for the name of the table is one I still stand by: it is a join between the jobseekers and vacancies tables. If the vacancies table is renamed I would be happy for it to change for consistency. 

The reason the relationship on the jobseekers table is called `saved_jobs` is multi-faceted (brevity, consistency with the feature and controller, and it is what it says it is - aka clearer to understand)

Reply:
After implementing the saved jobs dashboard I retract everything I said...

The table shall be known forever more as `saved_jobs`